apps/CricMyBrainAndroidApp/app/src/main/java/com/cricmybrain/cricmybrainandroidapp/LocalServerActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.cricmybrain.cricmybrainandroidapp;
+
+import android.app.Activity;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.KeyEvent;
+import android.webkit.WebView;
+
+public class LocalServerActivity extends Activity {
+    WebView myWebView;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_local_server);
+
+        myWebView = (WebView) findViewById(R.id.localServerWebview);
+        myWebView.setWebViewClient(new MyWebViewClient(LocalServerActivity.this));
+        myWebView.getSettings().setBuiltInZoomControls(true);
+        myWebView.getSettings().setSupportZoom(true);
+        myWebView.getSettings().setJavaScriptCanOpenWindowsAutomatically(true);
+        myWebView.getSettings().setJavaScriptEnabled(true);
+        myWebView.getSettings().setAllowFileAccess(true);
+        myWebView.getSettings().setDomStorageEnabled(true);
+        myWebView.loadUrl("http://10.0.0.117");
```

Comment:
We can do that, but this Local Activity is just for my test purpose in case our server goes down. Technically user should never give any website name in a text view or something. It should just work seamlessly. I just created this activity to test to make sure I can test all app functionalities locally. I can move this activity class to test folder I guess under ..\CricMyBrainAndroidApp\app\src\androidTest. Let me know your opinion on that ?